ci-operator/step-registry/ccm/conf/apply-feature-gate/ccm-conf-apply-feature-gate-commands.sh

```diff
@@ -23,8 +76,41 @@ spec:
   customNoUpgrade:
     enabled:
     - ExternalCloudProvider
+    - CSIMigrationAWS
   featureSet: CustomNoUpgrade
 EOF
+}
+
+function waitForKubeletAndKCMRollout() {
+  echo "$(date -u --rfc-3339=seconds) - Wait for some time for cluster operators to reconcile feature gate change"
+  sleep 5m
```

Comment:
Sleeping for 5m makes me a bit uncomfortable, can we not have some sort of loop where we check a condition say every 10s?

Reply:
I refactored the step to wait for --cloud-provider=external flag on kubelet configs first.